downloads/index.md

```diff
@@ -7,7 +7,7 @@
     <div class="container pt-sm-2">
       <div class="row" id="windows-instructions" style="display: none;">
         Install the latest Julia version (<a href="#current_stable_release">v{{stable_release}}</a>{{stable_release_date}}) from the <a href="https://www.microsoft.com/store/apps/9NJNWW8PVKMN">Microsoft Store</a> by running this in the command prompt:
-        <pre><code class="language-plaintext cmdprompt-block">winget install julia -s msstore</code></pre>
+        <pre><code class="language-plaintext cmdprompt-block">winget install --name Julia --id 9NJNWW8PVKMN -e -s msstore</code></pre>
```

Comment:
Are there long version also for the -e -s options?

Reply:
yes `--exact` and `--source`